package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
we already have a UI framework (`react-bootstrap`). I don't see a need for adding `@material-ui/core`